spec/application_helper_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+describe ApplicationHelper, type: :helper do
+  describe "#page_title" do
+    it "returns GOV.UK for a normal page" do
```

Comment:
This description doesn't feel correct for this test.

Looking at the code it seems like there should be a few it's:

- it joins a collection of title parts with ' - '
- it removes any empty title parts
- it prefixes a title with GOV.UK
- it prefixes the title with admin when the controller is prefixed with admin